Configurations/unix-Makefile.tmpl

```diff
@@ -741,62 +741,62 @@ lint:
+	( cd "$(SRCDIR)"; $(PERL) crypto/bn/bn_prime.pl > crypto/bn/bn_prime.h )
 
 generate_crypto_objects:
-	( cd $(SRCDIR); $(PERL) crypto/objects/objects.pl -n \
-				crypto/objects/objects.txt \
-				crypto/objects/obj_mac.num \
-				> crypto/objects/obj_mac.new && \
+	( cd "$(SRCDIR)"; $(PERL) crypto/objects/objects.pl -n \
+	                          crypto/objects/objects.txt \
+	                          crypto/objects/obj_mac.num \
+	                          > crypto/objects/obj_mac.new && \
 	    mv crypto/objects/obj_mac.new crypto/objects/obj_mac.num )
-	( cd $(SRCDIR); $(PERL) crypto/objects/objects.pl \
-				crypto/objects/objects.txt \
-				crypto/objects/obj_mac.num \
-				> include/openssl/obj_mac.h )
-	( cd $(SRCDIR); $(PERL) crypto/objects/obj_dat.pl \
-				include/openssl/obj_mac.h \
-				> crypto/objects/obj_dat.h )
-	( cd $(SRCDIR); $(PERL) crypto/objects/objxref.pl \
-				crypto/objects/obj_mac.num \
-				crypto/objects/obj_xref.txt \
-				> crypto/objects/obj_xref.h )
+	( cd "$(SRCDIR)"; $(PERL) crypto/objects/objects.pl \
+	                          crypto/objects/objects.txt \
+	                          crypto/objects/obj_mac.num \
+	                          > include/openssl/obj_mac.h )
+	( cd "$(SRCDIR)"; $(PERL) crypto/objects/obj_dat.pl \
+	                          include/openssl/obj_mac.h \
+	                          > crypto/objects/obj_dat.h )
+	( cd "$(SRCDIR)"; $(PERL) crypto/objects/objxref.pl \
+	                          crypto/objects/obj_mac.num \
+	                          crypto/objects/obj_xref.txt \
+	                          > crypto/objects/obj_xref.h )
 
 generate_crypto_conf:
-	( cd $(SRCDIR); $(PERL) crypto/conf/keysets.pl \
-			        > crypto/conf/conf_def.h )
+	( cd "$(SRCDIR)"; $(PERL) crypto/conf/keysets.pl \
+	                          > crypto/conf/conf_def.h )
 
 generate_crypto_asn1:
-	( cd $(SRCDIR); $(PERL) crypto/asn1/charmap.pl \
-			        > crypto/asn1/charmap.h )
+	( cd "$(SRCDIR)"; $(PERL) crypto/asn1/charmap.pl \
+	                          > crypto/asn1/charmap.h )
 
 generate_fuzz_oids:
-	( cd $(SRCDIR); $(PERL) fuzz/mkfuzzoids.pl \
-				crypto/objects/obj_dat.h \
-				> fuzz/oids.txt )
+	( cd "$(SRCDIR)"; $(PERL) fuzz/mkfuzzoids.pl \
+	                          crypto/objects/obj_dat.h \
+	                          > fuzz/oids.txt )
 
 # Set to -force to force a rebuild
 ERROR_REBUILD=
 errors:
-	( b=`pwd`; set -e; cd $(SRCDIR); \
-          $(PERL) util/ck_errf.pl -strict -internal; \
-          $(PERL) -I$$b util/mkerr.pl $(ERROR_REBUILD) -internal )
-	( b=`pwd`; set -e; cd $(SRCDIR)/engines; \
-          for E in *.ec ; do \
-              $(PERL) ../util/ck_errf.pl -strict \
-                -conf $$E `basename $$E .ec`.c; \
-              $(PERL) -I$$b ../util/mkerr.pl $(ERROR_REBUILD) -static \
-                -conf $$E `basename $$E .ec`.c ; \
-          done )
+	( b="`pwd`"; set -e; cd "$(SRCDIR)"; \
```

Comment:
For me the quotes around `pwd` do not seem necessary.

Reply:
Have you tried spaces in SRCDIR ?

Reply:
I only tried doing a build in a source directory with a space in it (`SRCDIR` is assigned `.` by `./Configure` though). (This seems to be the same case as above.)

Reply:
Ah, okay, then SRCDIR is "." as well.
But SRCDIR would have spaces if you do ../a b c/confg && build
but that fails completely, even with this PR

Reply:
Yes, it's supposed to. Rule definitions look like this in `Makefile`:
```make
crypto/rc4/rc4-%.s:>../x  y/crypto/rc4/asm/rc4-%.pl
```

Space should be escaped with a backslash in GNU Make rule headers (`targets : prerequisites` lines). With that fixed, the next issue is missing quoting in compiler invocation.
I'm looking into the latter.